doc/nrf/libraries/others/dm.rst

```diff
@@ -0,0 +1,75 @@
+.. _mod_dm:
+
+Distance Measurement
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Distance Measurement module provides an integration of :ref:`nrfxlib:nrf_dm` into |NCS|.
+
+Overview
+********
+
+The Distance Measurement module is independent of the communication protocol used.
+The :ref:`nrfxlib:mpsl` is used to access the radio module which is required by the :ref:`nrfxlib:nrf_dm` library.
+The distance is measured during the allocated timeslot.
+
+Using Distance Measurement
+**************************
+
+To use Distance Measurement module, enable it using the :kconfig:`CONFIG_NRF_DM` Kconfig option and initialize it in your :file:`main.c` file.
+After synchronizing the nodes between which the measurement is to be performed, add the measurement request.
+The callback function - :c:func:`data_ready` is called when the measurement data is available.
+
+Complete the following steps:
+
+1. Enable the :kconfig:`CONFIG_NRF_DM` Kconfig option.
+#. Include :file:`dm.h` in your :file:`main.c` file.
+#. Call :c:func:`dm_init()` to initialize the module.
+#. Call :c:func:`dm_request_add()` to perform the measurement.
+
+Configuration
+*************
+
+To use the Distance Measurement module, enable the :kconfig:`CONFIG_NRF_DM` Kconfig option.
+
+To adjust the synchronization of the nodes, change the values of the following options:
```

Comment:
I would focus here on the settings that the users are supposed to change and then maybe have it more scenario oriented: how do you increase scalability? How do you reduce latency? and so on.

Reply:
Done. I have added a description of the parameters.